src/ert/ensemble_evaluator/callbacks.py

```diff
@@ -57,19 +57,22 @@ def forward_model_ok(
                     # Already initialised, ignore
                     continue
 
-                grid = xtgeo.grid_from_file(ens_conf.grid_file)
-                props = xtgeo.gridproperty_from_file(
-                    pfile=file_path, name=config_node.getKey(), grid=grid
-                )
-
-                data = props.values1d.data
-                field_config = config_node.getFieldModelConfig()
-                trans = field_config.get_init_transform_name()
-                data_transformed = field_config.transform(trans, data)
-
-                run_arg.ensemble_storage.save_field_data(
-                    config_node.getKey(), run_arg.iens, data_transformed
-                )
+                try:
+                    grid = xtgeo.grid_from_file(ens_conf.grid_file)
+                    props = xtgeo.gridproperty_from_file(
+                        pfile=file_path, name=config_node.getKey(), grid=grid
+                    )
+
+                    data = props.values1d.data
+                    field_config = config_node.getFieldModelConfig()
+                    trans = field_config.get_init_transform_name()
+                    data_transformed = field_config.transform(trans, data)
+
+                    run_arg.ensemble_storage.save_field_data(
+                        config_node.getKey(), run_arg.iens, data_transformed
+                    )
+                except BaseException as ex:
```

Comment:
Is it possible to be more specific than BaseException?

Reply:
Question is if you want to. There are a number of things that can go wrong with loading the file and you might want to simply say that it failed rather than propagate an unexpected exception.